### ISSUE TITLE
Question about running project

### ISSUE THREAD

**Issue by Taimurkhan1999** [Selithrarion/quasar-nest_audio-clipper#1]
![Capture](https://github.com/Selithrarion/vue_audio-tools/assets/135840023/276d5b9c-bdeb-4318-b501-8c75b2df020f)

**Comment by Selithrarion** [comment]
@Taimurkhan1999 as i see u're trying to start instagram clone project, not the audio tools one. the error says u need to add .env file (based on .env.example). so just rename .env.example to .env. also there are some empty api/mail fields u need to fill but maybe it'll start without them idk

**Comment by Taimurkhan1999** [comment]
yes you are right, i have already tried it but can not run the project 
![Uploading image.png…]()

**Comment by Taimurkhan1999** [comment]
I am really interested in your projects please help me in running it

**Comment by Selithrarion** [comment]
@Taimurkhan1999 im glad u interested! but honestly i think these projects are so small so there is no sense in even launching them cuz i showed everything on gifs. i can try to help you but please recheck ur image above cuz it didnt load properly https://github.com/Selithrarion/vue_audio-tools/issues/1#issuecomment-1906970010

**Comment by Taimurkhan1999** [comment]
yeah these are simple but i want to check somethings in it, for that reason i want to lunch it 
![image](https://github.com/Selithrarion/vue_audio-tools/assets/135840023/f6bcf926-3a28-460e-9352-df2a78d8c1ab)
same issue with all of projects

**Comment by Selithrarion** [comment]
@Taimurkhan1999 are u sure u have "docker desktop" installed and launched? https://www.docker.com/products/docker-desktop/

**Comment by Taimurkhan1999** [comment]
![image](https://github.com/Selithrarion/vue_audio-tools/assets/135840023/910506b1-d34b-4032-8b50-5effcc03daaa)
![image](https://github.com/Selithrarion/vue_audio-tools/assets/135840023/93393853-4b39-4265-bd8e-39eded2bfe55)

**Comment by Taimurkhan1999** [comment]
![image](https://github.com/Selithrarion/vue_audio-tools/assets/135840023/173e03ec-3c08-4e79-84fc-07d13dde9602)

**Comment by Selithrarion** [comment]
@Taimurkhan1999 hi again! have u even tried to google this error? they say u need to add NODE_OPTIONS=--openssl-legacy-provider to .env or smth or downgrade ur nodejs to 16 major version https://stackoverflow.com/questions/69394632/webpack-build-failing-with-err-ossl-evp-unsupported

**Comment by Taimurkhan1999** [comment]
yes, I have but  the issue was something else. Thank you
